test/tests/kernels/diffusion.i

```diff
@@ -2,6 +2,8 @@
   type = MFEMMesh
   file = gold/mug.e
   dim = 3
+  serial_ref = 0
```

Comment:
To keep things consistent with native MOOSE, we probably would want support for using `uniform_refine` to set the (serial?) refinement level here. 

Since this is user facing, these options should probably be listed as `serial_refine` and `parallel_refine` to make it explicit for a user what they do.

Reply:
Ok so just for clarity, how should we treat `uniform_refine`? Is the serial refinement doubly defined as `serial_refine` and `uniform_refine`? Maybe we can just accept either variable but not both?

Reply:
Either but not both sounds sensible to me. The main motive is to make it straightforward for an existing MOOSE user to request refinement in the usual way

Reply:
Should be done now!